src/usdm-client.ts

```diff
@@ -497,7 +497,7 @@ export class USDMClient extends BaseRestClient {
     return this.getPrivate('fapi/v2/account');
   }
 
-  getAccountComissionRate(
+  getAccountCommissionRate(
     params: BasicSymbolParam,
   ): Promise<UserCommissionRate> {
```

Comment:
Something like this, so the old function is still available (and does the same as the new one) - it just warns the user for now that they should use the new one.

```
    /**
   * @deprecated Use getAccountCommissionRate() instead. This will be removed in the next major release.
   **/
  getAccountComissionRate(
    params: BasicSymbolParam,
  ): Promise<UserCommissionRate> {
    return this.getAccountCommissionRate(params);
  } 

  getAccountCommissionRate(
    params: BasicSymbolParam,
  ): Promise<UserCommissionRate> {
```

Reply:
Thank you for your thoughtful review! I’ve added the old function back with a deprecation warning.